public/js/actions/sources.js

```diff
@@ -51,6 +52,19 @@ function newSource(source) {
   };
 }
 
+function loadSourceMapSources(tab, source, dispatch) {
```

Comment:
This lives in an actions file but isn't an action creator. Don't worry about it for now, but every function in here should return something that is dispatched. But we may want to have sourcemap loading actions, as mentioned in my other comment, so this would become an action creator. Anyway, not a problem right now.